tests/interfaces.py

```diff
@@ -7515,6 +7516,68 @@ def test_AddUser_FormSubmit(self):
         utils.check_val_is_in("id=\"view_users_list\"", body)
         utils.check_val_is_in(self.test_user_name, body)
 
+    @runner.MAGPIE_TEST_USERS
+    def test_AddUser_FormSubmit_WithSupplementalUsernameRegex_Invalid(self):
+        """
+        Check that the supplemental_user_name_regex setting is used to validate a new user name when the user name is
+        invalid according to that regex but is valid according to the ax.PARAM_REGEX.
+        """
```

Comment:
All new tests should start with some form of `utils.warn_version(self, "<reason>", "3.37", skip=True)` to avoid remote tests failing on older instances. Could also add `.. versionadded:: 3.37` to their docstrings.